<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiHashes</class>
 <widget class="QWidget" name="UiHashes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hashes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="hashes_TabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_Hashes_SHA1">
      <attribute name="title">
       <string>SHA1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_20">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_Sha1InputLabel">
         <property name="text">
          <string>Input bytes (e.g. AA 11 22)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser_SHA1_InputBytes">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_44">
         <property name="text">
          <string>SHA1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA1_Output">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Hashes_SHA2">
      <attribute name="title">
       <string>SHA2</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_Sha2InputLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>HEX input: 0 bytes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser_SHA2_InputBytes">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>SHA2 224</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA2_224_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>SHA2 256</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA2_256_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>SHA2 384</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA2_384_Output">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>SHA2 512</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA2_512_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>SHA2 512/224</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA2_512_224_Output">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_SHA2_512_256">
         <property name="text">
          <string>SHA2 512/256</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA2_512_256_Output">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Hashes_SHA3">
      <attribute name="title">
       <string>SHA3</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_Sha3InputLabel">
         <property name="text">
          <string>Input bytes (e.g. AA 11 22)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser_SHA3_InputBytes">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>SHA3 224</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA3_224_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>SHA3 256</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA3_256_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>SHA3 384</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA3_384_Output">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>SHA3 512</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_SHA3_512_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Hashes_MD5">
      <attribute name="title">
       <string>MD5</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_21">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_Md5InputLabel">
         <property name="text">
          <string>Input bytes (e.g. AA 11 22)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser_MD5_InputBytes">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_45">
         <property name="text">
          <string>MD5</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_MD5_Output">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Hashes_KECCAK">
      <attribute name="title">
       <string>KECCAK</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_KeccakInputLabel">
         <property name="text">
          <string>Input bytes (e.g. AA 11 22)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser_KECCAK_InputBytes">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>KECCAK 224</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_KECCAK_224_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>KECCAK 256</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_KECCAK_256_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_36">
         <property name="text">
          <string>KECCAK 384</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_KECCAK_384_Output">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>KECCAK 512</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_KECCAK_512_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Blake">
      <attribute name="title">
       <string>BLAKE2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_Blake2InputLabel">
         <property name="text">
          <string>Input bytes (e.g. AA 11 22)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser_BLAKE2_InputBytes">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_47">
         <property name="text">
          <string>BLAKE 2B 160</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_BLAKE2B_160_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_48">
         <property name="text">
          <string>BLAKE 2B 256</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_BLAKE2B_256_OUTPUT_3">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_49">
         <property name="text">
          <string>BLAKE 2B 384</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_BLAKE2B_384_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_42">
         <property name="text">
          <string>BLAKE 2B 512</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_BLAKE2B_512_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string>BLAKE 2S 128</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_BLAKE2S_128_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_38">
         <property name="text">
          <string>BLAKE 2S 160</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_BLAKE2S_160_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_41">
         <property name="text">
          <string>BLAKE 2S 224</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEdit_BLAKE2S_224_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_37">
         <property name="text">
          <string>BLAKE 2S 256</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="lineEdit_BLAKE2S_256_OUTPUT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
